gulpfile.js

```diff
@@ -11,7 +11,7 @@ var typescript = require('typescript');
 var mocha = require('gulp-mocha');
 
 gulp.task('check-format', function() {
-  return gulp.src(['*.js', 'lib/**/*.ts', 'test/**/*.ts'])
+  return gulp.src(['*.js', 'lib/**/*.ts', 'test/**/*.ts', '!test/fixtures/**/*.ts'])
```

Comment:
The serializer depends on the whitespace determinism of generated d.ts files, so disabling formatting reflects this. For example, the formatter makes short declarations like `interface A { b: number; }` one line, which never happens in a generated d.ts file.